angular-client/src/index.html

```diff
@@ -8,6 +8,6 @@
     <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.1/mapbox-gl.css" rel="stylesheet" />
   </head>
   <body>
-    <app-context />
+    <app-context></app-context>
```

Comment:
Custom Elements are not allowed to be self-closed. `<app-context />` was an invalid syntax, and after migration, this causes the build to fail